packages/plugins/src/plugins/everclear/positionFetcher.ts

```diff
@@ -0,0 +1,54 @@
+import { NetworkId } from '@sonarwatch/portfolio-core';
+import { getAddress, pad } from 'viem';
+import { Cache } from '../../Cache';
+import { Fetcher, FetcherExecutor } from '../../Fetcher';
+import {
+  everclearToken,
+  platformId,
+  STAKING_CONTRACT_ADDRESS,
+} from './constants';
+import { getEvmClient } from '../../utils/clients';
+import { abi } from './abi';
+import { ElementRegistry } from '../../utils/elementbuilder/ElementRegistry';
+import { zeroBigInt } from '../../utils/misc/constants';
+
+const NETWORK_ID = NetworkId.everclear;
+const executor: FetcherExecutor = async (owner: string, cache: Cache) => {
+  const client = getEvmClient(NETWORK_ID);
+  const formattedAddress = getAddress(owner);
+  // pad to bytes32
+  const paddedAddress = pad(formattedAddress, { size: 32 });
+
+  const [amount, lockedUntil] = await client.readContract({
+    abi,
+    address: STAKING_CONTRACT_ADDRESS,
+    functionName: 'positionData',
+    args: [paddedAddress],
+  });
+
+  if (amount === zeroBigInt) return [];
+
+  const registry = new ElementRegistry(NETWORK_ID, platformId);
+
+  const liquidity = registry.addElementLiquidity({
+    label: 'Staked',
+  });
+
+  liquidity.addLiquidity().addAsset({
+    address: everclearToken.address,
+    amount: amount.toString(),
+    attributes: {
+      lockedUntil: Number(lockedUntil * 1000n), // convert to ms
+    },
+  });
+
+  return registry.getElements(cache);
+};
+
+const fetcher: Fetcher = {
+  id: `${platformId}-positions`,
+  networkId: NetworkId.everclear,
```

Comment:
Leverage our local constant ?

```suggestion
  networkId: NETWORK_ID,
```